components/load_from_hf_hub/fondant_component.yaml

```diff
@@ -23,4 +23,9 @@ args:
   n_rows_to_load:
     description: Optional argument that defines the number of rows to load. Useful for testing pipeline runs on a small scale
     type: int
-    default: None
\ No newline at end of file
+    default: None
+  dataset_length:
```

Comment:
I would avoid having the user specify the length of the dataset and read it from the hf metadata directly